yoda.js

```diff
@@ -0,0 +1,56 @@
+$(document).ready(function(){
+
+  function replaceText( text ){
+    $("#yodaOutput").text(text)
+  }
+
+  function ellipses(){
+    replaceText("...")
+  }
```

Comment:
Instead of "ellipse", how about "awkwardSilence", "pondering", or similar?  The ellipse represents something.